examples/base/params.py

```diff
@@ -0,0 +1,101 @@
+# bluemira is an integrated inter-disciplinary design tool for future fusion
+# reactors. It incorporates several modules, some of which rely on other
+# codes, to carry out a range of typical conceptual fusion reactor design
+# activities.
+#
+# Copyright (C) 2021 M. Coleman, J. Cook, F. Franza, I.A. Maione, S. McIntosh, J. Morris,
+#                    D. Short
+#
+# bluemira is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# bluemira is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with bluemira; if not, see <https://www.gnu.org/licenses/>.
+
+"""
+An example of how to use Parameters and ParameterFrames within bluemira.
+"""
+
+# %%[markdown]
+# # Parameters and ParameterFrames in bluemira
+# `Parameters` and `ParameterFrames` are the mechanism bluemira uses to contain
+# metadata about a given value.
+# Each `Parameter` must have a unit associated with the value and can have a
+# source, description and long_name.
+#
+# The mechanics of the unit system in bluemira are fairly staight forward
+# It provides an implicit interface to convert units to the
+# [base units of bluemira](
+# https://bluemira.readthedocs.io/en/latest/conventions.html#unit-convention).
```

Comment:
I'm not sure what you mean here. Maybe something like this?

> \`ParameterFrame\`s implicitly convert parameter values' units to the \[base units of bluemira\](
https://bluemira.readthedocs.io/en/latest/conventions.html#unit-convention).

Reply:
Sure :)